extras/ams_wrapper/src/preprocessing/preprocess_image.py

```diff
@@ -106,6 +106,7 @@ def preprocess_binary_image(image: bytes, channels: int = None,
         if scale:
             image_array = image_array * scale
     except Exception as e:
+        print(str(e))
```

Comment:
With logging facilities merged to ams branch, we can now use `logger.exception()` here instead of `print()`

Reply:
Done.